airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1760 @@
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: google.cloud.dlp_v2.types.DeidentifyTemplate
+        """
+
+        client = self.get_conn()
+
+        if organization_id:
+            parent = DlpServiceClient.organization_path(organization_id)
+        elif project_id:
+            parent = DlpServiceClient.project_path(project_id)
+        else:
+            raise AirflowException(
+                "Please provide either organization_id or project_id."
+            )
+
+        return client.create_deidentify_template(
+            parent=parent,
+            deidentify_template=deidentify_template,
+            template_id=template_id,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def create_dlp_job(
+        self,
+        project_id=None,
+        inspect_job=None,
+        risk_job=None,
+        job_id=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+        wait_until_finished=True
+    ):
+        """
+        Creates a new job to inspect storage or calculate risk metrics.
+
+        :param project_id: (Optional) Google Cloud Platform project ID where the
+            DLP Instance exists. If set to None or missing, the default
+            project_id from the GCP connection is used.
+        :type project_id: str
+        :param inspect_job: (Optional) The configuration for the inspect job.
+        :type inspect_job: dict or google.cloud.dlp_v2.types.InspectJobConfig
+        :param risk_job: (Optional) The configuration for the risk job.
+        :type risk_job: dict or google.cloud.dlp_v2.types.RiskAnalysisJobConfig
+        :param job_id: (Optional) The job ID.
+        :type job_id: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :param wait_until_finished: (Optional) If true, it will keep polling the job state
+            until it is set to DONE.
+        :type wait_until_finished: bool
+        :rtype: google.cloud.dlp_v2.types.DlpJob
+        """
+
+        client = self.get_conn()
+
+        if not project_id:
```

Comment:
It's dead code, because decorator `fallback_to_default_project_id` prevent to execute this part of code